Cannot find module 'bn.js' error when requiring after clean install

To reproduce:
1) Run `npm init -y` and `npm install rlp`
2) In that directory, run `node` and try to import `rlp` module with `require('rlp')`
3) Following error appears:
~~~bash
$ node
> require('rlp')
Error: Cannot find module 'bn.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
>
~~~

This is probably due to `bn.js` listed as a devDependency in package.json, instead of as a regular dependency. Moving `bn.js` to dependencies should fix this error.

--- a/package.json
+++ b/package.json
@@ -46,7 +46,8 @@
     "url": "https://github.com/ethereumjs/rlp/issues"
   },
   "dependencies": {
-    "safe-buffer": "^5.1.1"
+    "safe-buffer": "^5.1.1",
+    "bn.js": "^4.11.1"
   },
   "devDependencies": {
     "@ethereumjs/config-nyc": "^1.0.0",
@@ -55,8 +56,7 @@
     "@ethereumjs/config-tslint": "^1.0.0",
     "@types/bn.js": "^4.11.3",
     "@types/mocha": "^5.2.5",
-    "@types/node": "^10.12.2",
-    "bn.js": "^4.11.1",
+    "@types/node": "^10.12.2",    
     "coveralls": "^2.11.4",
     "mocha": "4.1.0",
     "nyc": "^13.1.0",

